force-app/main/default/lwc/reviewSessions/reviewSessions.js

```diff
@@ -65,8 +81,16 @@ export default class ReviewSessions extends LightningElement {
         return this.labels.totalSessions + ": " + this._serviceSessions.length;
     }
 
+    get sessionStartFieldName() {
+        return this._serviceScheduleModel.sessionFields.sessionStart.apiName;
+    }
+
+    get sessionEndFieldName() {
+        return this._serviceScheduleModel.sessionFields.sessionEnd.apiName;
+    }
+
     setLabels() {
-        this.objectName = this._serviceScheduleModel.labels.serviceSession.objectPluralLabel;
+        this.objectName = this._serviceScheduleModel.labels.serviceSession.objectApiName;
```

Comment:
this was being used to display "x sessions" so we probably need another property for this.pluralobjectlabel or something?